containers/bash/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:latest
```

Comment:
Also same as in the other case, you could use the pre-built `mcr.microsoft.com/vscode/devcontainers/base:alpine` image if you want. I think all the dependencies in the Dockerfile are covered, so you'd be set.

Reply:
Updated.